.github/workflows/dev_deploy.yml

```diff
@@ -65,3 +65,12 @@ jobs:
       environment: "${{ inputs.environment }}"
       ref: "${{ needs.setup-environment.outputs.git-hash }}"
     secrets: inherit
+
+  run-qa-suite:
+    needs: [run-deployment]
+    if: ${{ (inputs.environment == 'perf') || (inputs.environment == 'staging') || (inputs.environment == 'dev') }}
```

Comment:
Was the tested? Github action boolean checks are notoriously difficult to get right.

Reply:
This used to work when we had it running before (same code just adds dev). And Lucas did test this (see description).

Reply:
Sorry, talking specifically about the `if` block evaluating to `true` when it should not. I can see the happy path was tested, did we do any testing to show the logic works as expected in the `false` case? 

If the dev check is removed and we run this action, does it skip this step? Boolean Github Actions checks are finicky and we don't want it to try and run the regression against prod (even if the QA-repo won't permit it).

Reply:
We did [a test here,](https://github.com/department-of-veterans-affairs/notification-api/actions/runs/7507607593/job/20441494354) and the conditional worked thankfully.